File: trainning1team/src/main/java/com/bnksystem/trainning1team/dto/Equip/EquipDetailResponse.java
package com.bnksystem.trainning1team.dto.Equip;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class EquipDetailResponse {
    private String serialNO;
    private String eqNM;
    private String typeNM;
    private String eqModel;
    private String eqMaker;
    private String detail;
    private String manualImgUrl;
    private String statusNM;
}


/*
- 장비 시리얼 번호
- 장비명
- 제품 종류
- 모델명
- 제조사
- 상세정보 : 특이사항
- 메뉴얼 img
- 출고상태

 */

File: GearTracker/app/src/main/java/com/bsys/geartracker/data/model/response/EquipDetailResponse.kt
package com.bsys.geartracker.data.model.response

data class EquipDetailResponse(
    val serialNO: String,
    val eqNM: String,
    val typeNM: String,
    val eqModel: String,
    val eqMaker: String,
    val detail: String,
    val manualImgUrl: String,
    val statusNM: String
) {
    override fun toString(): String {
        return "EquipDetailResponse(serialNO='$serialNO', eqNM='$eqNM', typeNM='$typeNM', eqModel='$eqModel', eqMaker='$eqMaker', detail='$detail', manualImgUrl='$manualImgUrl', statusNM='$statusNM')"
    }
}
